c2corg_ui/templates/article/helpers/view.html

```diff
@@ -49,30 +49,34 @@
 </%def>
 
 <%def name="get_article_associated_users(article)">\
-  % if article['associations']['users']:
-    % if len(article['associations']['users']) > 1 or not article['associations']['users'][0]['document_id'] == article['author']['user_id']:
-      <div class="name-icon-value" ng-click="detailsCtrl.openTab($event)">
-        <h4><span translate>associated users</span><span class="glyphicon glyphicon-menu-right"></span></h4>
-        <div class="name-icon">
-          <span class="icon-user"></span>
-          <p translate>associated users</p>
-        </div>
-        <span class="detail-text accordion">
-          % if article.get('associations').get('users'):
-            <p>
-              % for user in article['associations']['users']:
-                <%
-                    user_id = user['document_id']
-                    user_lang = user['locales'][0]['lang']
-                    is_last_associated_user = loop.last
-                %>
-                <a href="${request.route_path('profiles_view', id=user_id, lang=user_lang)}">${user['name']}</a>${'' if is_last_associated_user else ', '}
-              % endfor
-            </p>
-          % endif
+  % if article.get('associations'):
+    % if article['associations']['users']:
+      % if len(article['associations']['users']) > 1:
+        <div class="name-icon-value" ng-click="detailsCtrl.openTab($event)">
+          <h4><span translate>associated users</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+          <div class="name-icon">
+            <span class="icon-user"></span>
+            <p translate>associated users</p>
+          </div>
+          <span class="detail-text accordion">
+            % if article.get('associations').get('users'):
+              <p>
+                % for user in article['associations']['users']:
+                  <%
+                      user_id = user['document_id']
+                      user_lang = user['locales'][0]['lang']
+                      is_last_associated_user = loop.last
```

Comment:
Do you really need this intermediary var? Using ``loop.last`` seems fairly understandable in the code below :)